js/foundation.tabs.js

```diff
@@ -365,6 +371,7 @@ class Tabs {
       }
     }
 
+    $(window).off('popstate', this._checkDeepLink);
```

Comment:
No check for `this.options.deepLink` ?

Reply:
One year later: _that was a stupid idea_.

Options can changes, listeners can be created in complex tests, remplaced, created dynamically. Here we only want to clear everything. No need for that check.

Reply:
True. So do we need further changes here?

Reply:
No, don't worry. I seen that while working on https://github.com/zurb/foundation-sites/pull/11077